pkg/reconciler/instances/ory/hydra/hydra_test.go

```diff
@@ -0,0 +1,187 @@
+package hydra
+
+import (
+	"context"
+	k8smocks "github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes/mocks"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+	appv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/utils/pointer"
+	"testing"
+	"time"
+)
+
+const (
+	testNamespase = "kyma-system"
+)
+
+func Test_TriggerSynchronization(t *testing.T) {
+	logger := zaptest.NewLogger(t).Sugar()
+	t.Parallel()
+	t.Run("Should trigger synchronization if hydra-maester is behind hydra", func(t *testing.T) {
+		// given
+		hydraStartTimePod1 := time.Date(2021, 10, 10, 10, 10, 10, 10, time.UTC)
```

Comment:
These names do not really resemble the expected time. How about renaming them to the more meaningful names - `hydraLatestStartTime`, `hydraMiddleStartTime` and `hydraEarliestStartTime`?